Modules/trigger.py

```diff
@@ -35,22 +35,47 @@ def __init__(self, bot: pydle.BasicClient, nickname: str, target: str,
         self.identified = identified
 
     @classmethod
-    def from_bot_user(cls, bot: pydle.BasicClient, nickname: str, target: str):
+    def from_bot_user(cls, bot: pydle.BasicClient, nickname: str, target: str)\
+            -> 'Trigger':
         """
         Creates a `Trigger` object from a user dictionary as used by pydle.
-        :param bot: Instance of the bot.
-        :param nickname: Command sender's nickname.
-        :param target: The message target (usually a channel or, if it was sent in a query window, the bot's nick)
+
+        Args:
+            bot (pydle.BasicClient): instance of the bot
+            nickname (str): Command sender's nickname.
+            target (str): The message target  (usually a channel or, if it was
+                sent in a query window, the bot's nick)
+
+        Returns:
+            Trigger
         """
         user = bot.users[nickname]
         return cls(bot, user["nickname"], target,
-                   ident=user["username"], hostname=user["hostname"], away=user["away_message"],
-                   account=user["account"], identified=user["identified"])
+                   ident=user["username"], hostname=user["hostname"],
+                   away=user["away_message"], account=user["account"],
+                   identified=user["identified"])
 
     @property
-    def channel(self):
+    def channel(self)->str or None:
+        """
+        The channel the trigger came from.
+
+        Returns:
+            str : channel, if it exists
+        """
         return self.target if self.bot.is_channel(self.target) else None
 
     async def reply(self, msg: str):
-        """Sends a message in the same channel or query window as the command was sent."""
-        await self.bot.message(self.channel if self.channel else self.nickname, msg)
+        """
+        Sends a message in the same channel or query window as the command was
+        sent.
+
+        Args:
+            msg (str): message to send
+
+        Returns:
```

Comment:
1. I believe a one-liner is more than sufficient here (or a one-sentencer, since 79 lines >_<)
2. Can we remove the empty Returns:?

Reply:
uh... i *may* have been overzealous with this one. will revert.